frontend/src/scenes/insights/InsightDisplayConfig.tsx

```diff
@@ -120,44 +125,68 @@ export function InsightDisplayConfig({ filters, activeView, disableTable }: Insi
                         <RetentionDatePicker />
                         <RetentionReferencePicker />
-                    </>
+                    </span>
                 )}
 
                 {showPathOptions && (
-                    <span className="filter">
+                    <span className="space-x-2 flex items-center text-sm">
                         <PathStepPicker />
                     </span>
                 )}
 
                 {showComparePrevious[activeView] && (
-                    <span className="filter">
+                    <span className="space-x-2 flex items-center text-sm">
                         <CompareFilter />
                     </span>
                 )}
             </div>
-            <div className="display-config-inner-row">
+            <div className="flex items-center space-x-2 flex-wrap mx-0 my-2">
+                {activeView === InsightType.TRENDS && (
+                    <span className="space-x-2 flex items-center text-sm">
+                        <span>y-axis format</span>
+                        <LemonSelect
+                            value={filters.y_axis_format || 'numeric'}
+                            onChange={(value) => {
+                                if (isYAxisFormat(value)) {
+                                    setFilters({ y_axis_format: value })
+                                }
+                            }}
+                            bordered
+                            dropdownPlacement={'bottom-end'}
+                            dropdownMatchSelectWidth={false}
+                            data-attr="chart-y-axis-format"
+                            disabled={!canFormatAxis(filters.display)}
+                            options={yAxisFormatSelectOptions}
+                            type={'stealth'}
+                            size={'small'}
+                        />
+                    </span>
+                )}
                 {showChartFilter(activeView) && (
-                    <span className="filter">
+                    <span className="space-x-2 flex items-center text-sm">
                         <span className="head-title-item">Chart type</span>
                         <ChartFilter filters={filters} disabled={filters.insight === InsightType.LIFECYCLE} />
                     </span>
                 )}
+
                 {showFunnelBarOptions && filters.funnel_viz_type === FunnelVizType.Steps && (
                     <>
-                        <span className="filter">
+                        <span className="space-x-2 flex items-center text-sm">
                             <FunnelDisplayLayoutPicker />
                         </span>
                     </>
                 )}
                 {showFunnelBarOptions && filters.funnel_viz_type === FunnelVizType.TimeToConvert && (
-                    <span className="filter">
+                    <span className="space-x-2 flex items-center text-sm">
```

Comment:
So, are we resolving to in-line classnames as a general rule? I approve

Reply:
Yeah, I had to fix spacing. So thought I'd fix it properly :)

Reply:
Not saying we have to but in the case of repeated styles, that could be pulled out to a variable `rowClassNames` to dry it up